DataHandler.py

```diff
@@ -392,45 +391,23 @@ def extract_mod_data_to_json(folder_path: str) -> list[Any]:
 
                     # Process each mod_data block
                     for mod_data_content in matches:
+                        mod_data_match = yaml.safe_load(file_content)
+                        mod_data_content = mod_data_match.get("Hatsune Miku Project Diva Mega Mix+", {}).get("megamix_mod_data")
 
-                        data_dict = get_dict(mod_data_content, False)
-
-                        if data_dict == "":
+                        if mod_data_content == "":
```

Comment:
You want to check if it's `None` here too, since the last get would return that if the key is missing. `if not mod_data_content` would check for both `""` and `None`